### DISCUSSION TITLE
Has anyone had any experience with an Itron G5R1 meter

### DISCUSSION THREAD

**Discussion by pfletch101** [bemasher/rtlamr#308] [Q&A]
My utility has just installed these in my neighborhood. The model is listed on the Google Docs list (with an ERT type of 0!?), but I can't see any transmissions from it (or, indeed, anything else within range). The manufacturers' blurb on it rather suggests that it uses a different (i.e. non-ERT) technology ('Generation 5 IOT Network') to communicate with the utility. Does anyone have any wisdom to offer?

Edited to add some additional information: there is still no traffic if I default to the SCM protocol, but I do see some reasonably regular transmissions - apparently from water meters (probably mine and one other) - using the R900 protocol. This would seem to rule out a hardware or software setup problem.

![20240807_151821](https://github.com/user-attachments/assets/2c122ca7-d63c-48d9-b8eb-d5bc9aaba51d)

**Comment by alwoody** [comment]
Could be one of the type that does not operate in (bubble up mode) it just answers when queried . Sent from my iPhoneOn Aug 7, 2024, at 2:18 PM, pfletch101 ***@***.***> wrote:﻿
My utility has just installed these in my neighborhood. The model is listed on the Google Docs list (with an ERT type of 0!?), but I can't see any transmissions from it (or, indeed, anything else within range). The manufacturers' blurb on it rather suggests that it uses a different (i.e. non-ERT) technology ('Generation 5 IOT Network') to communicate with the utility. Does anyone have any wisdom to offer?
20240807_151821.jpg (view on web)

**Reply by pfletch101** [reply]
I don't think that is the explanation, since the utility is advertising the availability of real-time data from the meters to customers on its website, and I can't see how you would easily do that unless the meter was transmitting it spontaneously at relatively frequent intervals. It is possible (since the meter was only installed last week, and I can't yet see any evidence of the availability of data from it on my account page) that it hasn't been activated. I have messaged the support folks for an ETA for the web-based data, and will recheck for transmissions when that is available,

**Comment by pfletch101** [comment]
I can now see my hourly usage, albeit a couple of days in arrears, by logging into my account on the NYSEG website, so the meter has obviously been activated and is transmitting data. I still cannot see any transmissions from it on the 900 MHz band. From the (rather limited!) technical information about this meter that I can find on the web, however, it appears to be able to communicate **both** in the 'old' 900 MHz band **and** in the 2.4 GHz band, which is outside the range covered by inexpensive SDRs. My guess is that NYSEG are using the latter band, either for valid technical reasons (?better range, less interference) or to prevent customers from doing what we are trying to do - the latter would be fairly typical of them.

**Comment by dcorwin8222** [comment]
Any more work on this? Would like to get the same info from my NYSEG meter as well.

**Reply by pfletch101** [reply]
I am afraid we are out of luck! Further spelunking in what documentation is available on these meters suggests that NYSEG are, indeed, using the meter's 2.4 GHz transmission band, which, as I noted above, is out of the reception range for standard SDRs. It is possible to use a down-converter to bring the 2.4 GHz signal within a normal SDR's frequency range, but you would still have to figure out NYSEG's (almost certainly) proprietary encoding schema.

**Comment by ajvpb** [comment]
Just to reiterate interest, I have the same meter from Pacificorp and would love to crack it. I can pick up dozens of meters with rtlamr but none fit my 9 digit ID number. I see someone listed this in the unofficial .csv so that gives me hope!

**Comment by jwaters23** [comment]
Ugh, has a perfectly working meter with RTL433 beautifully into home assistant. Stupid eversource “upgraded” me to this smart meter yesterday and now I see the model isn’t working with RTL. 

Would love to see how to get this going. Happy to help with providing anything I can.
